src/SFA.DAS.Apim.Developer.Web/SFA.DAS.Apim.Developer.Web.csproj

```diff
@@ -47,6 +47,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\..\..\das-shared-packages\SFA.DAS.DfESignIn.Auth\SFA.DAS.DfESignIn.Auth\SFA.DAS.DfESignIn.Auth.csproj" />
```

Comment:
This doesn't seem correct 😄

Reply:
Hi Dan, as a result of the library not being nuged'd this is the only at present to reference the library